MaplePacketLib2/Tools/PacketWriter.cs

```diff
@@ -181,6 +181,12 @@ public void WriteUnicodeString(string value = "")
     public void WriteRawUnicodeString(string value)
     {
         int length = value.Length * 2;
+
+        if (length == 0)
```

Comment:
You could just check for `value.Length` being `0` first to avoid an unnecessary multiplication